mockwebserver/src/main/kotlin/mockwebserver3/MockWebServer.kt

```diff
@@ -187,7 +187,7 @@ class MockWebServer : Closeable {
       field = protocolList
     }
 
-  private var started: Boolean = false
+  var started: Boolean = false
```

Comment:
This is a public API change - to a new class.

Reply:
Needs a whole bunch of @Synchronized things to make this public.

Reply:
Is this public for end users or just public for testing?

Reply:
I'll revert.